docs/apps/ipsec-offload.md

```diff
@@ -47,7 +48,7 @@ to load the hardware FXP pipeline with the IPsec package.
 Follow the instructions in
 [Running infrap4d](/guides/es2k/running-infrap4d.md)
 and prepare the system with generated TDI.json and context.json file references.
-In order to  offload IPsec, fixed function support must be enabled in infrap4d. 
+In order to  offload IPsec, fixed function support must be enabled in infrap4d.
```

Comment:
The implication of this sentence is that fixed-function support is an optional feature that the user can enable or disable.

If this is the case, then it should say what the user needs to do to enable it.

If this is not the case, then the sentence should be rewritten to say what it means, or it should be removed.

Reply:
Yes, this is an optional part of the config file that only applies to IPsec portion. There is a blank `fixed_function` section in the original es2k_skip_p4.conf file, which needs to be edited.

What this section needs to enable this feature is described in following lines (lines 56-64)